### PR TITLE
Re-enable touchpoints and update form

### LINKED ISSUE #4309: Update Touchpoints design to make it clear that issues needing resolution should be directed to support
### Background
Looking at our Touchpoints feedback from the last couple weeks, there are several instances of users requesting help on a specific issue (i.e., "I can't log in" "can't get to my results" etc). These issues should be directed to the support inbox since Touchpoints feedback isn't monitored regularly.

### Action requested
Update the Touchpoints designs to make the support inbox/support help pages a more prominent.

### Acceptance Criteria
New designs for Touchpoints have been created.

### Additional context
We have two Touchpoints forms: one for the app and one for the static site. 

Current static site form 
Screen Shot 2022-09-21 at 3 16 35 PM
Screen Shot 2022-09-21 at 3 16 41 PM

Current app form
Screen Shot 2022-09-21 at 3 17 21 PM

### PR DESCRIPTION
**Pull request by emyl3**
# FRONTEND PULL REQUEST

## Related Issue

Resolves #4309 

## Changes Proposed
- Change the content of the form to make it clearer it is not for support help

## Additional Information
- Added a wiki page on how to[ update touchpoints](https://github.com/CDCgov/prime-simplereport/wiki/Update-Touchpoints) 
- Thank you to @fzhao99 for all his help with this 😸 

## Testing
- Go to dev4 and ensure you see the updated touchpoints form in-app and can submit feedback

## Screenshots / Demos
**Form**
Screen Shot 2022-12-15 at 17 04 37

**Feedback submitted**
Screen Shot 2022-12-15 at 17 04 54

### CODE REVIEW

**Review by fzhao99 — APPROVED** [review]
lgtm!

**Review by johanna-skylight — APPROVED** [review]
LGTM

File: frontend/public/touchpoints.js [hunk 1]
```diff
@@ -656,169 +652,127 @@ var formOptions = {
   modalButtonText: "How can we improve this site?",
 };
 
+// Note: When updating Touchpoints you need to update this HTML
 // To whoever has to maintain this, I'm sorry this is so gross.
```
[the remark this answers is not in the document]
Comment (emmastephenson) [reply frontend/public/touchpoints.js:656]: It's definitely something to think about, but IIRC the benefit to using this raw HTML form is that Touchpoints itself takes care of placing the button appropriately. There are definitely other ways to implement it, but most of those require you to design a custom button. Definitely something to look into next year, though.

**Review by emmastephenson — DISMISSED** [review]
LGTM, thank you for re-enabling this Elisa! 🙌

**Comment by kenieh** [comment]
LGTM! Pretty sure I know the answer to this, but is there the ability to adjust spacing? I'd just want to move the "Provide feedback  on SimpleReport" header down more to be closer to the question below. The header as it is right now is a bit too high up, creating clutter and distracting from the important message above about reaching out to support fo rhelp

**Comment by emyl3** [comment]
> LGTM! Pretty sure I know the answer to this, but is there the ability to adjust spacing? I'd just want to move the "Provide feedback on SimpleReport" header down more to be closer to the question below. The header as it is right now is a bit too high up, creating clutter and distracting from the important message above about reaching out to support fo rhelp

Hi Kenny! I was playing around with the form a bit. I think I can do that. How much spacing did you want for the top? I can also make the "Please provide feedback on SimpleReport" an `<h3>` to make it more visually distinct as well as add some spacing to the top of it. :)

**Review by kenieh — APPROVED** [review]

**Review by fzhao99 — APPROVED** [review]

**Pull request merged**